README.md

```diff
@@ -565,6 +565,13 @@ Takes a run with dark images for use in pedestals, and posts to the elog.
     </td>
 </tr> 
 
+<tr>
+    <td>verify_hutch</td>
+    <td>
+usage: Verifies that the passed argument is a known hutch, exit 0 for true or exit for false.
```

Comment:
Copy and pasting from your usage within the script (and changing `${BASH_SOURCE[0]}` to `verify_hutch`
```suggestion
usage: verify_hutch <hutch>
 Script that returns confirmation if the passed hutch is a known/existing hutch, success indicated by exit 0, failure by exit 1. 
```